### ISSUE TITLE
undefined reference to `wifi_station_clear_enterprise_identity'

### ISSUE THREAD

**Issue by claudiosousa** [martin-ger/esp_peap_psk#1]
I've built esp-open-sdk, but when trying to 'make' esp_peap_psk, I'm getting "undefined reference to `wifi_station_clear_enterprise_identity'"

D build/app.out
build/app_app.a(user_main.o):(.text+0x2c): undefined reference to `wifi_station_clear_enterprise_identity'
build/app_app.a(user_main.o): In function `wifi_handle_event_cb':
/home/claudio/Documents/git/esp_peap_psk/user/user_main.c:22: undefined reference to `wifi_station_clear_enterprise_identity'
build/app_app.a(user_main.o):(.irom0.text+0x40): undefined reference to `wifi_station_set_enterprise_identity'
build/app_app.a(user_main.o): In function `user_init':
/home/claudio/Documents/git/esp_peap_psk/user/user_main.c:106: undefined reference to `wifi_station_set_enterprise_identity'
collect2: error: ld returned 1 exit status
Makefile:119: recipe for target 'build/app.out' failed
make: *** [build/app.out] Error 1

Can't see wifi_station_clear_enterprise_identity defined nor esp-open-sdk on nor on esp_peap_psk

**Comment by bonnefoi** [comment]
The same for me...

**Comment by martin-ger** [comment]
You need to install the latest NONOS-SDK 2.1.0  (see esp-open-sdk Makefile), there it is included:
https://github.com/espressif/ESP8266_NONOS_SDK/blob/master/include/wpa2_enterprise.h

**Comment by notdeclan** [comment]
I have the NONOS SDK 2.1.0 installed however I still have this error.

**Comment by martin-ger** [comment]
Did you change the reference in the beginning of the Makefile in the root directory of the esp-open-sdk?
```
# Vendor SDK version to install, see VENDOR_SDK_ZIP_* vars below
# for supported versions.
VENDOR_SDK = 2.1.0
```

**Comment by notdeclan** [comment]
Yes, and I have tried rebuilding the SDK on another machine, both with the VENDOR_SDK set to 2.1.0

**Comment by martin-ger** [comment]
Please check with "nm libwpa2.a | grep enterprise_identity", whether the file ".../esp-open-sdk/sdk/lib/libwpa2.a" contains:
```
00000010 T wifi_station_clear_enterprise_identity
...
00000024 T wifi_station_set_enterprise_identity

```
If not, you have for some reasons not the latest libwpa2.a, if yes, you are probably linking the wrong lib?

**Comment by notdeclan** [comment]
Fixed this issue by using the version **2.1.0-18-g61248df** in the ["./esp-open-sdk/Makefile"](https://github.com/pfalcon/esp-open-sdk/blob/master/Makefile) Makefile

`VENDOR_SDK = 2.1.0-18-g61248df`
